src/psygnal/qt.py

```diff
@@ -0,0 +1,76 @@
+"""Module that provides Qt-specific functionality for psygnal.
+
+This module provides convenience functions for starting and stopping a QTimer that
+will monitor "queued" signals and invoke their callbacks.  This is useful when
+psygnal is used in a Qt application, and you'd like to emit signals from a thread
+but have their callbacks invoked in the main thread.
+"""
+from __future__ import annotations
+
+from threading import Thread, current_thread
+
+from ._queue import emit_queued
+
+try:
+    from qtpy.QtCore import Qt, QTimer
+except ImportError:
```

Comment:
If I correctly remeber for part of `qtpy` (it looks like even `2.3.0`, when the latest is `2.3.1`) lack of Qt binding is signaled by `RuntimeError`, not `ImportError`